pyspectral/tests/test_reflectance.py

```diff
@@ -221,3 +215,33 @@ def test_reflectance(self):
             self.assertTrue(hasattr(refl, 'compute'))
         except ImportError:
             pass
+
+
+def test_get_as_array_from_scalar_input():
+    """Test the function to return an array when input is a scalar"""
+    res = get_as_array(1)
+
+    if hasattr(res, 'compute'):
+        assert res.compute()[0] == 1
+    else:
+        assert res[0] == 1
+
+
+def test_get_as_array_from_numpy_array_input():
+    """Test the function to return an array when input is a numpy array."""
+    res = get_as_array(np.array([1.0, 2.0]))
+
+    if hasattr(res, 'compute'):
+        np.testing.assert_allclose(res.compute(), np.array([1.0, 2.0]), 5)
+    else:
+        np.testing.assert_allclose(res, np.array([1.0, 2.0]), 5)
+
+
+def test_get_as_array_from_list_input():
+    """Test the function to return an array when input is a list."""
+    res = get_as_array([1.0, 2.0])
+
+    if hasattr(res, 'compute'):
+        np.testing.assert_allclose(res.compute(), np.array([1.0, 2.0]), 5)
+    else:
+        np.testing.assert_allclose(res, np.array([1.0, 2.0]), 5)
```

Comment:
And again, this pure-Numpy functionality should be tested every time.

Reply:
Ok, thanks @pnuu 
But right now I don't manage to make a numpy version with your suggestion above that tests the `numpy.array` function. It keeps testing the built-in function instead!?
I changed your example above to `new=np.array`!